static_report/src/sdlc/generate-e2e-data.sh

```diff
@@ -1,29 +1,46 @@
 #!/bin/bash
 # For generating comparison_data.json from `piperider-getting-started`
 # NOTE: This file assumes you are calling from the root of repo
+# NOTE: Pass parameters to shell call e.g. $1
+if [ $2 = "clean" ]; then
+	rm -rf piperider-getting-started
+fi
 
-# clone repo
-git clone https://github.com/InfuseAI/piperider-getting-started.git
+# First 2 run.jsons
```

Comment:
```suggestion
# First 2 run.json
```

Reply:
should be plural, since it's two jsons, not one :)

Reply:
Sorry for my poor English.